packages/fsweb/polyfills/document.js

```diff
@@ -2,7 +2,8 @@ export function createElement() {
   return {
     setAttribute: function() {},
     pathname: '',
-    style: {}
+    style: {},
+    nodeType: 1
```

Comment:
Portal component is crashing unless the nodeType is one of the allowed. 1 means normal dom element (html tag).